### ISSUE TITLE
Assumption of Spatial Heterogeneity As Not Random; Does this necessarily entail choice or adaptation?

### ISSUE THREAD

**Issue by tamos** [uchicago-computation-workshop/luis_bettencourt#4]
Set up: The assumption made in the quoted text below is because spatial heterogeneity has persisted across neighbourhoods through time and social change, it is not a random phenomenon. 

**Question(s):** 
a) Why does this necessarily entail individual agency (choice) or response to stimuli (adaptation)?
You address some of this in your text and the supplementary materials, but additional explanation would be beneficial for my understanding.

b) In your opinion, is this the complete set of possible explanations? Are there alternatives you chose to exclude? Why?

c) Is social change (and its components, such as demographic turnover) not partially defined by the revitalization or impoverishment of certain areas?  What effect would this have on your assumptions? 

I am thinking of the following examples: The move to suburbs in the 20th century revitalized small towns around cities while hollowing out the inner-city; Detroit/Windsor/Pittsburgh are undergoing racial, economic, and other types of social change as they are "revitalizing".

**Section: Results
Subsection: Information and Spatial Selection**

**Relevant text:** 

> As an illustration of the problem, consider the complex pattern of household income in New York City neighborhoods (Figure 1A); see Figs. S1-S17 for other cities. We observe strong heterogeneity at different spatial scales, from adjacent neighborhoods with different average household incomes to larger recognizable patches of wealth and poverty, e.g. the Upper East Side (the richest part of Manhattan) or the Bronx (generally rather poor), Fig. 1B. As is well known, this spatial heterogeneity is long-lived, persisting for decades or longer, through many economic cycles and substantial demographic turnover: thus, these patterns are not the result of accidental fluctuations, but rather of decades of choice and adaptation by successive waves of individuals.
> Such rich and detailed patterns contrasts with the simple normal distribution for (the logarithm of) household income across the entire metropolitan area, Fig. 1C. ...

**Comment by khan1792** [comment]
A great question. I'm also thinking about to what extent the author's bio-ecological approach can be applied to the human society. Hope you can ask the author this general question along with your detailed ones.

**Comment by lmbett** [comment]
Yes to your questions: there is both churn of people, and change in people who are residents. This is very important in practice, but hard to separate in the data... there may be places where it becomes possible to study, which would be great.
